### ISSUE TITLE
GPL dependency

### ISSUE THREAD

**Issue by TripleWhy** [Firwood-Software/AdvanceDLSupport#66]
### Description

This library, or at least the NuGet package has a dependency to StrictEmit, which is GPLv3 licensed. Are you actually sure that you can release AdvanceDLSupport as LGPL in that case?

**Comment by Nihlus** [comment]
Yes. I am the author of that library as well, and I've granted myself a product-specific LGPL-compatible license of StrictEmit to ADL :P

**Comment by TripleWhy** [comment]
I see, thanks for the response. But should that not be noted somewhere then? Because If I want to use ADL, or another library that depends on ADL, I will have to distribute a copy of StrictEmit as well.
Also note that VS makes me accept the GPL when I try to install ADL or something dependent on it from NuGet.

**Comment by DylanFPS** [comment]
This actually sparked a discussion in the Firwood official Discord; Nihlus said he's going to add a license grant saying that if you only use ADL (and not use StrictEmit directly,) you can use ADL. If you wish to use ADL in a non-GPL project, you'll have to get in touch with Nihlus to negotiate a similar grant for your project.

TL;DR: There's gonna be a funky license that lets you use StrictEmit through ADL.

**Comment by Nihlus** [comment]
@TripleWhy Ah, I was not aware that VS did that. As Dylan says, I'll be looking at being more explicit about this.
